Changelog/index.html

```diff
@@ -27,20 +27,20 @@ <h1>Equilibrium Balance Mod Changelog</h1>
         <p><div><span title="Click to expand" class="SpoilerDarkCollapsed"><b>Last patch changes</b></span>
                 <p><h2>Version 62</h2>
                 
-                <p> In this version we focus to fix stuff that break while new balance patches was coming out. 
-                    Another focus was make faction diversity deeper and promote more sharp desing, where not everything is poolish and smooth but have more spice inside. 
-                    In the last there was eco movement where very sliglthli promote eco T2 tech, where T3 eco take slightli longer to build. 
+                <p> In this version, we focus to fix stuff that breaks while new balance patches were coming out. 
+                    Another focus was to make faction diversity deeper and promote more sharp design, where not everything is poolish and smooth but have more spice inside. 
+                    In the last, there was eco-movement where very slightly promote T2 eco-tech, while T1 is easier to destroy while T3 eco take slightly longer to build. 
                 </p>
                 
                 <p><b>Misc</b>
```

Comment:
This whole block has inconsistent capitalisation: I would start each bullet point with capital letter